tsconfig.json

```diff
@@ -20,5 +20,9 @@
     "incremental": true,
     "typeRoots": ["./node_modules/@types", "./custom_typing"]
   },
-  "include": ["**/*.ts", "**/*.tsx"]
+  "include": [
+    "**/*.ts",
+    "**/*.tsx",
+    "./src/styles/Terminal/atom-one-light-custom.js"
```

Comment:
This happens when you change a js file to ts or something to that effect. It's annoying, but RM this change, since there is no need for this.

Reply:
I was modeling this file after the ./node_modules/react-syntax-highlighter/dist/cjs/styles/hljs files. These files are originally *.js but are defined as typescript objects via ./node_modules/@types/react-syntax-highlighter/index.d.ts. I didn't try changing it to ts but I can look into it since its only one file that's being imported.